apps/tunes/serializers.py

```diff
@@ -22,6 +22,12 @@ class Meta:
         fields = ('artist', 'name', 'genre', 'code')
 
 
+class LastPlaylistSerializer(serializers.Serializer):
```

Comment:
Let's write unit tests for this to ensure a response can be serialized with and without a context